projects/packages/jetpack-mu-wpcom/webpack.config.js

```diff
@@ -1,10 +1,12 @@
 const path = require( 'path' );
 const jetpackWebpackConfig = require( '@automattic/jetpack-webpack-config/webpack' );
 const verbumConfig = require( './verbum.webpack.config.js' );
+const moduleConfig = require( './webpack.config.modules.js' );
```

Comment:
It's a bit annoying that the two config files have different naming schemes. That said, I prefer the format you've listed (which matches better what we have elsewhere), and the other one is preexisting.